sys/can/conn/isotp.c

```diff
@@ -174,7 +174,7 @@ int conn_can_isotp_send(conn_can_isotp_t *conn, const void *buf, size_t size, in
                 if (msg.content.ptr == conn) {
                     ret = -EIO;
                 }
-                /* No break */
+                /* Fall through */
```

Comment:
Shouldn't this be `Falls through`?

Reply:
According to gcc documentation
- `-Wimplicit-fallthrough=0` disables the warning altogether.
- `-Wimplicit-fallthrough=1` matches `.*` regular expression, any comment is used as fallthrough comment.
- `-Wimplicit-fallthrough=2` case insensitively matches `.*falls?[ \t-]*thr(ough|u).*` regular expression.
- `-Wimplicit-fallthrough=3` case sensitively matches one of the following regular expressions:
  - `-fallthrough`
  - `@fallthrough@`
  - `lint -fallthrough[ \t]*`
  - `[ \t.!]*(ELSE,? |INTENTIONAL(LY)? )?`
  - `FALL(S | |-)?THR(OUGH|U)[ \t.!]*(-[^\n\r]*)?`
  - `[ \t.!]*(Else,? |Intentional(ly)? )?`
  - `Fall((s | |-)[Tt]|t)hr(ough|u)[ \t.!]*(-[^\n\r]*)?`
  - `[ \t.!]*([Ee]lse,? |[Ii]ntentional(ly)? )?`
  - `fall(s | |-)?thr(ough|u)[ \t.!]*(-[^\n\r]*)?`
- `-Wimplicit-fallthrough=4` case sensitively matches one of the following regular expressions:
  - `-fallthrough`
  - `@fallthrough@`
  - `lint -fallthrough[ \t]*`
  - `[ \t]*FALLTHR(OUGH|U)[ \t]*`
- `-Wimplicit-fallthrough=5` doesn’t recognize any comments as fallthrough comments, only attributes disable the warning.

So the 's' doesn't matter.
Btw, `-Wextra` enables `-Wimplicit-fallthrough=3` by default.

Reply:
> Btw, -Wextra enables -Wimplicit-fallthrough=3 by default.

that's what I wasn't sure about! Thx for clarifying! 

Re-ACK

Reply:
Ok. Then I don't care (though we used "Falls through" in other places).